charts/rancher-wins-upgrader/0.0.100/Chart.yaml

```diff
@@ -3,6 +3,7 @@ annotations:
   catalog.cattle.io/namespace: cattle-wins-system
   catalog.cattle.io/os: windows
   catalog.cattle.io/release-name: rancher-wins-upgrader
+  catalog.cattle.io/rancher-version: '>= 2.5.0-0 <= 2.5.99-0'
```

Comment:
Can you change this to `'>= 2.5.0-0 < 2.6.0-0'`

Reply:
Updated the annotation. Can you please review again?